### PR TITLE
chore(rxNotify): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
[FRMW-357](https://jira.rax.io/browse/FRMW-357)

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxNotify/scripts/rxNotify.js [hunk 1]
```diff
@@ -1,188 +1,4 @@
...
- * Displays all messages in a stack.
- *
- * @param {String=} [stack='page'] The message stack to associate with
- *
- * @example
- * <pre>
- * <rx-notifications stack="myCustomStack"></rx-notifications>
- * </pre>
- */
-.directive('rxNotifications', function (rxNotify) {
```
Comment (CITguy) [inline src/rxNotify/scripts/rxNotify.js]: moved to `src/rxNotify/scripts/rxNotifications.js`

File: src/rxNotify/scripts/rxNotify.js [hunk 2]
```diff
@@ -1,188 +1,4 @@
...
- * Display a static message with styling taken from `rx-notifications`.
- *
- * @param {String=} [type='info'] The type of notification (e.g. 'warning', 'error')
- *
- * @example
- * <pre>
- * <rx-notification type="warning">This is a message!</rx-notification>
- * </pre>
- */
-.directive('rxNotification', function (rxNotify) {
```
Comment (CITguy) [inline src/rxNotify/scripts/rxNotify.js]: moved to `src/rxNotify/scripts/rxNotification.js`

File: src/rxNotify/scripts/rxNotify.js [hunk 3]
```diff
@@ -533,138 +349,4 @@ angular.module('encore.ui.rxNotify', ['ngSanitize', 'ngAnimate'])
...
- * @example
- * <pre>
- * rxPromiseNotifications.add($scope.deferred.promise, {
- *     loading: 'Loading Message',
- *     success: 'Success Message',
- *     error: 'Error Message'
- * });
- * </pre>
- */
-.factory('rxPromiseNotifications', function (rxNotify, $rootScope, $q, $interpolate) {
```
Comment (CITguy) [inline src/rxNotify/scripts/rxNotify.js]: moved to `src/rxNotify/scripts/rxPromiseNotifications.js`

**Comment by bry4n** [comment]
LGTM

**Comment by 100stacks** [comment]
@CITguy there is more to `rxNotify` for tracking notifications than I would have expected.  LGTM :+1:

**Pull request merged**